holidays/countries/ivory_coast.py

```diff
@@ -0,0 +1,128 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+from gettext import gettext as tr
+
+from holidays import FEB
+from holidays.groups import (
+    ChristianHolidays,
+    IslamicHolidays,
+    InternationalHolidays,
+    StaticHolidays,
+)
+from holidays.observed_holiday_base import ObservedHolidayBase, SUN_TO_NEXT_MON
+
+
+class IvoryCoast(
+    ObservedHolidayBase, ChristianHolidays, IslamicHolidays, InternationalHolidays, StaticHolidays
+):
+    """Ivory Coast holidays.
+
+    References:
+        * <https://natlex.ilo.org/dyn/natlex2/r/natlex/fe/details?p3_isn=44374>
+        * <https://www.droit-afrique.com/uploads/RCI-Decret-1996-205-jours-feries.pdf>
+        * <https://fr.wikipedia.org/wiki/F%C3%AAtes_et_jours_f%C3%A9ri%C3%A9s_en_C%C3%B4te_d%27Ivoire>
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Ivory_Coast>
+        * <https://www.timeanddate.com/holidays/ivory-coast/>
+        * <https://holidayapi.com/countries/ci/>
+    """
+
+    country = "CI"
+    # %s (estimated).
+    estimated_label = tr("%s (estimé)")
+    # %s (observed).
+    observed_label = tr("%s (observé)")
+    # %s (observed, estimated).
+    observed_estimated_label = tr("%s (observé, estimé)")
+    # Ivory Coast gained independence in 1960.
+    start_year = 1960
+    default_language = "fr"
```

Comment:
```suggestion
    default_language = "fr"
    # %s (estimated).
    estimated_label = tr("%s (estimé)")
    # %s (observed).
    observed_label = tr("%s (observé)")
    # %s (observed, estimated).
    observed_estimated_label = tr("%s (observé, estimé)")
    # Ivory Coast gained independence in 1960.
    start_year = 1961
```
It's better to start with a formally full year. But it would be even better to take a year for which we know about the existence of legal acts (if we find Decree No. 65-127 from 1965-04-02 with all subsequent amendments, it will be 1966; if we base on Decree No. 93-321 from 1993-03-11, it will be 1994, etc.)

Reply:
I had tried to find the 1965 decree but was unable to find it. Should I change the start_year to 1994?

Reply:
I checked the decree again, the earliest one that I referred to and could find even now is from 1996. https://www.droit-afrique.com/upload/doc/cote-divoire/RCI-Decret-1996-205-jours-feries-chomes.pdf

Hence, changed the start_year to 1997.